### ISSUE TITLE
Incompatible with ember-cli-node-assets

### ISSUE THREAD

**Issue by Turbo87** [embroider-build/embroider#500]
It seems that addons using ember-cli-node-assets are currently incompatible with embroider. I've seen this on other addons in the past too, and currently I'm running into it with https://github.com/shipshapecode/ember-flatpickr:

```
Build Error (WaitForTrees)

Cannot find module './vendor/flatpickr/flatpickr.js' from '/private/var/folders/jb/3qmm1y051xdcv33pjkh4b01c0000gn/T/embroider/42038c/node_modules/@embroider/synthesized-vendor'
```

/cc @shipshapecode @ef4

**Comment by ef4** [comment]
I have seen it work, but maybe something in either embroider or the addon changed.

**Comment by GCheung55** [comment]
@Turbo87 did you exclude `flatpickr` in `autoImport` config? That's what I needed to do to get things building again.

**Comment by Turbo87** [comment]
hmm... I'm not importing `flatpickr` myself, only using it through the components provided by the addon. Shouldn't ember-auto-import ignore imports from addons that don't bring it as a dependency? 🤔

**Comment by GCheung55** [comment]
@Turbo87 Maybe, but how should ember-auto-import know?

**Comment by ef4** [comment]
I continue to be unable to reproduce this problem. I see ember-cli-node-assets working correctly under embroider in several apps. Can anybody share a reproduction where it does not?

**Comment by Turbo87** [comment]
the flatpickr addon is no longer using `ember-cli-node-assets` since https://github.com/shipshapecode/ember-flatpickr/pull/535. you might need to use an older release, I guess.

**Comment by ef4** [comment]
Fixed in #607
